package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "WTraining",
+  "version": "0.1.0",
+  "description": "WTraining",
+  "engines": {
+    "node": "10.14.1",
+    "npm": "6.4.1"
+  },
+  "scripts": {
+    "console": "node console.js",
+    "cover": "npm run test -- --coverage",
+    "test": "NODE_ENV=testing jest --runInBand --forceExit --detectOpenHandles",
+    "test-inspect": "NODE_ENV=testing node --inspect --debug-brk jest",
+    "eslint-check": "eslint --print-config .eslintrc.js --ignore-pattern ./.eslintrc.js | eslint-config-prettier-check",
+    "lint": "eslint \"**/*.js\" --ignore-pattern ./.eslintrc.js",
+    "lint-diff": "git diff --diff-filter=ACM --name-only --cached --relative | grep \\\\.js$ | xargs eslint",
+    "lint-fix": "npm run lint -- --fix",
+    "outdated": "npm outdated --depth 0",
+    "pretest": "npm run lint",
+    "start-dev": "nodemon --inspect server.js",
+    "prestart-dev": "npm run lint",
+    "migrations": "sequelize db:migrate",
```

Comment:
Add this script here `"migrations-test": "NODE_ENV=testing sequelize db:migrate"`